locales/de/filters.json

```diff
@@ -91,6 +91,10 @@
 		"filter.23.name": "AdGuard Ukrainischer Filter",
 		"filter.23.description": "Filter, der Anzeigen auf ukrainischen Webseiten sperrt."
 	},
+	{
+		"filter.24.name": "Quick Fixes-Filter",
```

Comment:
Error?
```suggestion
		"filter.24.name": "AdGuard Quick Fixes Filter",
```

Reply:
if so, should be fixed in crowdin

Reply:
fixed in crowdin, but a hyphen is also needed

Reply:
@BlazDT Is it correct for German?